install/operator/pkg/generator/generator_test.go

```diff
@@ -78,7 +107,7 @@ func TestGenerator(t *testing.T) {
 		},
 	}
 
-	configuration, err := configuration.GetProperties("../../build/conf/config.yaml", context.TODO(), fake.NewFakeClient(), syndesis)
+	configuration, err := configuration.GetProperties("../../build/conf/config.yaml", context.TODO(), fakeClient(), syndesis)
```

Comment:
This fake.NewFakeClient() is not enough to allow for DeploymentConfig() to be handled.